### PR TITLE
*: add a limitation about not supporting `NULLIF()` and supporting `CASE`

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/18677
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: generated-columns.md [hunk 1]
```diff
@@ -154,4 +154,5 @@ The current limitations of JSON and generated columns are as follows:
 - You can neither convert a stored generated column to a normal column through the `ALTER TABLE` statement nor convert a normal column to a stored generated column.
 - You cannot modify the expression of a stored generated column through the `ALTER TABLE` statement.
 - Not all [JSON functions](/functions-and-operators/json-functions.md) are supported;
+- The [NULLIF functions](/control-flow-functions#nullif.md) is not supported. You can use the [CASE function](/control-flow-functions#case.md)as an alternative.
```
Comment (hfxsd) [inline generated-columns.md:157]:
```suggestion
- The [`NULLIF()` function](/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function](/control-flow-functions.md#case) as an alternative.
```

File: generated-columns.md [hunk 2]
```diff
@@ -154,4 +154,5 @@ The current limitations of JSON and generated columns are as follows:
 - You can neither convert a stored generated column to a normal column through the `ALTER TABLE` statement nor convert a normal column to a stored generated column.
 - You cannot modify the expression of a stored generated column through the `ALTER TABLE` statement.
 - Not all [JSON functions](/functions-and-operators/json-functions.md) are supported;
+- The [`NULLIF()` function](/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function](/control-flow-functions.md#case) as an alternative.
```
Comment (hfxsd) [inline generated-columns.md:157]:
```suggestion
- The [`NULLIF()` function](/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function](/control-flow-functions.md#case) instead.
```

**Review by hfxsd — APPROVED** [review]

File: generated-columns.md [hunk 3]
```diff
@@ -154,4 +154,5 @@ The current limitations of JSON and generated columns are as follows:
 - You can neither convert a stored generated column to a normal column through the `ALTER TABLE` statement nor convert a normal column to a stored generated column.
 - You cannot modify the expression of a stored generated column through the `ALTER TABLE` statement.
 - Not all [JSON functions](/functions-and-operators/json-functions.md) are supported;
+- The [`NULLIF()` function](/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function](/control-flow-functions.md#case) instead.
```
Comment (hfxsd) [inline generated-columns.md:157]:
```suggestion
- The [`NULLIF()` function]((/functions-and-operators/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function]((/functions-and-operators/control-flow-functions.md#case) instead.
```

File: generated-columns.md [hunk 4]
```diff
@@ -154,4 +154,5 @@ The current limitations of JSON and generated columns are as follows:
 - You can neither convert a stored generated column to a normal column through the `ALTER TABLE` statement nor convert a normal column to a stored generated column.
 - You cannot modify the expression of a stored generated column through the `ALTER TABLE` statement.
 - Not all [JSON functions](/functions-and-operators/json-functions.md) are supported;
+- The [`NULLIF()` function]((/functions-and-operators/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function]((/functions-and-operators/control-flow-functions.md#case) instead.
```
Comment (hfxsd) [inline generated-columns.md:157]:
```suggestion
- The [`NULLIF()` function](/functions-and-operators/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function]((/functions-and-operators/control-flow-functions.md#case) instead.
```

File: generated-columns.md [hunk 5]
```diff
@@ -154,4 +154,5 @@ The current limitations of JSON and generated columns are as follows:
 - You can neither convert a stored generated column to a normal column through the `ALTER TABLE` statement nor convert a normal column to a stored generated column.
 - You cannot modify the expression of a stored generated column through the `ALTER TABLE` statement.
 - Not all [JSON functions](/functions-and-operators/json-functions.md) are supported;
+- The [`NULLIF()` function](/functions-and-operators/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function]((/functions-and-operators/control-flow-functions.md#case) instead.
```
Comment (lilin90) [inline generated-columns.md:157]:
```suggestion
- Not all [JSON functions](/functions-and-operators/json-functions.md) are supported.
- The [`NULLIF()` function](/functions-and-operators/control-flow-functions.md#nullif) is not supported. You can use the [`CASE` function](/functions-and-operators/control-flow-functions.md#case) instead.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**